code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -13,6 +13,12 @@
 	var/move_delay = 0
 	var/egg_cooldown = 0
 
+/obj/structure/closet/cardboard/initialize()
+	if(prob(10))
+		var/summon_path = pick(/obj/item/clothing/head/collectable/bandana, /obj/item/clothing/under/sneaking_suit, /obj/item/clothing/mask/cigarette/cigar, /obj/item/robot_parts/l_arm)
```

Comment:
if you *really* want to get that serious, you can add worn_color as a system to items, all it would do is set ```Image.color = the_item.color```, then everyone can RP Big Boss :tm: